schemas/core/components/terms.json

```diff
@@ -17,26 +17,35 @@
       },
       "required": ["startTime", "endTime"]
     },
-    "fareRate": {
-      "description": "Booking fare rate",
-      "type": "object",
-      "properties": {
-        "amount": {
-          "type": "number",
-          "minimum": 0,
-          "multipleOf": 0.01
+    "fareRates": {
+      "type": "array",
+      "items": {
+        "description": "Booking fare rate",
+        "type": "object",
+        "properties": {
+          "amount": {
+            "type": "number",
+            "minimum": 0,
+            "multipleOf": 0.01
+          },
+          "currency": {
+            "$ref": "./units.json#/definitions/currency"
+          },
+          "timeInterval": {
```

Comment:
`timeInterval` or `timeUnit` ?

Reply:
`timeInterval` as unit is always in seconds and not selectable.